index.js

```diff
@@ -18,7 +18,7 @@ const platformLib = (() => {
 		}
 
 		case 'android': {
-			if (process.env.PREFIX !== '/data/data/com.termux/files/usr') {
+			if (process.env.TERMUX__PREFIX === undefined) {
```

Comment:
I don't think it's 2 underscores. Please make sure what's correct.

```suggestion
			if (process.env.TERMUX_PREFIX === undefined) {
```